go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/IBM/ibm-storage-odf-block-driver
 go 1.18
 
 require (
-	github.com/IBM/ibm-storage-odf-operator v1.4.0
+	github.com/IBM/ibm-storage-odf-operator v1.3.1-0.20230821004148-24b0497905fe
```

Comment:
how did the version go backwards?

Reply:
the first (old line) 1.4.0 is a tag 
the second (new line) is a commit number go is calculating and generating on it's own.
It didn't go backwards but that's how go decided to call this tag... we don't have impact on it. 

On official release we will always put the official tag and not a commit / branch name